docs/building-the-candidate-app/frontend-app-separation.md

```diff
@@ -0,0 +1,15 @@
+# Frontend app separation
+
+The frontend apps for voters and candidates (and VAA admins in the future) use the same `frontend` module with lots of shared components, utilities and styling. However, the routes and certain functions of the Candidate App are best kept separate from the voter app.
+
+## Routes
+
+Use the `/candidate` route for the Candidate App. Note that the root folder may change in the future to, e.g. `candidate-app`, so make any references to that either based on a shared constant or use `$page.url` to build routes.
+
+## Stores and other utilities
+
+Use the `$candidate` and `$voter` custom import paths for imports that are exclusively for use in either app. These map to `/src/lib/candidate` and `/src/lib/voter` respectively. Note that these folders may be later moved out of `/src/lib`, so make sure to always use the shorthand paths, e.g. `$candidate` instead of `$lib/candidate`.
+
+## Translations
+
+For the time being, use the `$_(stringPath)` function for all localizable texts, i.e. anything shown to the user. Place any strings related to the Candidate App under the key `candidateApp` and separate texts per view beneath it.
```

Comment:
Don't we want to encourage using the Strapi labels we have created?

Reply:
That would be nice, but we don't have those labels yet in the Strapi content model, I was thinking this is the easier path. But we could ofc ask them to create another block in `App Labels` for the Candidate App. Should we do that?

Reply:
I think it may be good for them to get their hands on Strapi 😃. Therefore, we can ask them to create the labels in Strapi.